gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddOrEditGradeItemPanel.html

```diff
@@ -5,9 +5,6 @@
 	<wicket:panel>
 
         <form wicket:id="addOrEditGradeItemForm" class="form-horizontal">
-		
-			<h2 wicket:id="heading">Add/Edit Title Here</h2>
-
             <div wicket:id="addGradeFeedback"></div>
```

Comment:
All headers have been removed from the panels and are set on the window itself.  Thankfully all of these panels are only ever used in the context of a window.